ocrd_models/ocrd_models/ocrd_file.py

```diff
@@ -61,6 +61,13 @@ def __str__(self):
         ])
         return '<OcrdFile ' + props + ']/> '
 
+    def __eq__(self, other):
+        print(self, other)
+        return self.ID == other.ID and \
+               self.url == other.url and \
+               EXT_TO_MIME[MIME_TO_EXT[self.mimetype]] == EXT_TO_MIME[MIME_TO_EXT[other.mimetype]] and \
+               self.fileGrp == other.fileGrp
```

Comment:
I wonder why we should go beyond `ID` here at all. METS file IDs are unique, and we cannot test for equality of the METS reference anyway.

Reply:
You're probably right, I'll keep the other attribute checks commented out though, in case we need more complex `==` chcecks in the future.